src/main/scala/com/raphtory/core/components/partition/BatchWriter.scala

```diff
@@ -92,10 +95,12 @@ class BatchWriter[T: ClassTag](
     logger.trace(s"Partition $partitionID: Received EdgeDelete message '$update'.")
     storage.timings(update.updateTime)
     storage.removeEdge(update.updateTime, update.srcId, update.dstId)
+    PartitionTelemetry.batchWriterEdgeDeletions.inc()
   }
 
   def printUpdateCount() = {
     processedMessages += 1
+    PartitionTelemetry.batchWriterGraphUpdates.inc()
```

Comment:
Is there any overhead to adding these? If so counting twice may not be the best idea when we can add up in graphana.

Reply:
Removing to avoid overheads